R/plot_convehull.r

```diff
@@ -0,0 +1,108 @@
+#' Function for plotting convex hulls.
+#'
+#' This function allows for data, that can be from Opta or Statsbomb, to be used
+#' for plotting convex hulls on top of an outline of a football pitch.
+#'
+#' @param data Dataframe that houses pass data. Opta dataframe must contain atleast the following columns: `x`, `y`, `finalX`, `finalY`, `playerId`.
+#' @param data_type Type of data that is being put in: opta or statsbomb. Default set to "statsbomb".
+#' @param colour The colour of the outline of the convex hull.
+#' @param title_plot Title of the plot.
+#' @param theme Indicates what theme the map must be shown in: dark (default), white, rose, almond.
+#' @return a ggplot2 object
+#'
+#' @import dplyr
+#' @import tidyr
+#' @import ggplot2
+#' @import ggsoccer
+#' @import purrr
+#'
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' plot <- plot_convexhull(data = data, data_type = "opta", colour = "blue", title_plot = "Team 1")
+#' plot
+#' }
+
+plot_convexhull <- function(data, data_type = "statsbomb", 
+                            colour = "#E74C3C", title_plot = "", theme = "dark") {
+  
+  if(data_type == "opta") {
```

Comment:
for `if` functions, please put a space between it and the opening parenthesis 

(there are several other cases of this inside the function)